### DISCUSSION TITLE
What is the best way to give credits to the software you've provided?

### DISCUSSION THREAD

**Discussion by zziime** [Shin-NiL/Godot-Android-Admob-Plugin#124] [General]
We are about to launch a game which we've been developing for the past two years. This game has an "about" section where we give the credit to people who've helped, I'm not sure how I should credit your plugin (name, social network, GitHub directory redirect). Let me know the best way, please.
Thanks! =)

**Comment by Shin-NiL** [comment]
Hello there my friend, I think "Shin-NiL" is fine and if is there a hyperlink the [github project](https://github.com/Shin-NiL/Godot-Android-Admob-Plugin) page is ok.

Thank you very much and don't forget to tell us when the game is published ;)
